ffc-pay-alerts/notify/notify-template/get-template-id.js

```diff
@@ -0,0 +1,9 @@
+const templateSchema = require('../../schema/event-schema')
+const { notifyEmailTemplateId } = require('../../config')
+
+const getTemplateId = (message) => {
+  const eventMatch = templateSchema.find(x => x.name === message?.name)
+  return eventMatch ? eventMatch.notifyTemplateId : notifyEmailTemplateId

```

Comment:
Question for my understanding, what is the default template we're using here?  Just a generic something is wrong and you need to investigate alert to us?

Reply:
Template is like the logic, hence, should be the same for all environment and with the same id. The environmental variable we need to consider in this case is the delivery/recipient address which has been taken care on another ticket.

The default template here is the one and only existing template that we use for all events at the moment.